roles/openshift_sanitize_inventory/tasks/unsupported.yml

```diff
@@ -18,3 +18,49 @@
     msg: |-
       The NetworkManager hook is considered a critical part of the DNS
       infrastructure.
+
+- set_fact:
+    __using_dynamic: True
+  when:
+  - hostvars[inventory_hostname][item] in ['dynamic']
+  with_items:
+  - "{{ hostvars[inventory_hostname] | vars_with_pattern(pattern='openshift_.*_storage_kind') }}"
```

Comment:
this is failing

Reply:
I think we can just omit most if not all the content in this file.